v3apps.go

```diff
@@ -255,3 +255,31 @@ func (c *Client) GetV3AppEnvironment(appGUID string) (V3AppEnvironment, error) {
 
 	return result, nil
 }
+
+type V3EnvVar struct {
+	Var map[string]*string `json:"var"`
```

Comment:
That's unfortunate the value is a pointer to string, but unfortunately this is technically correct according to the API docs.